src/store-api/src/region_request.rs

```diff
@@ -539,6 +539,16 @@ pub enum AlterKind {
         /// Name of columns to drop.
         names: Vec<String>,
     },
+    /// Set column default value.
+    SetDefaults {
+        /// Columns to change.
+        columns: Vec<SetDefault>,
+    },
+}
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct SetDefault {
+    pub name: String,
+    pub default_constraint: Vec<u8>,
```

Comment:
I think we should deserialize the constraint in try from.
```suggestion
    pub default_constraint: ColumnDefaultConstraint,
```

Reply:
Miss this comment? @linyihai

Reply:
@evenyag @linyihai

All the other comments were addressed, what about this one? Any suggestions?

Reply:
This is not a blocker. Parsing the ColumnDefaultConstraint requires the table schema, which is unavailable here.